### DISCUSSION TITLE
s3cmd

### DISCUSSION THREAD

**Discussion by Michael-Amann** [nginxinc/nginx-s3-gateway#44] [Q&A]
Hello,

Is it possible to use s3cmd with nginx-s3-gateway? https://github.com/s3tools/s3cmd

Thank and regards,
Michael

**Comment by dekobon** [comment]
Hi @Michael-Amann ,

My apologies for the late reply. This project does not provide an S3 interface, so you cannot use it with s3cmd. What this project does is front an S3 bucked and present the data from S3 as if it was being served on a normal web server. Ideally, to your end users they would not know that the data is coming from S3.
